tests/jballerina-unit-test/src/test/resources/test-src/expressions/access/field_access_negative.bal

```diff
@@ -376,3 +376,25 @@ function testInvalidBoundMethodAccessWithRemoteMethod(ServiceClass a,
 
     _ = e.ser.fn;
 }
+
+function testInvalidXMLMapFieldAccess1() {
```

Comment:
One minor change, add `returns error?` here. Because you have used check in the function.

Reply:
Do the same for other places.